subsquid/src/server-extension/resolvers/totalVolume.ts

```diff
@@ -0,0 +1,84 @@
+import {
+  Arg,
+  Field,
+  InputType,
+  Int,
+  ObjectType,
+  Query,
+  Resolver,
+} from "type-graphql";
+import type { EntityManager } from "typeorm";
+import { IsDateString, Min } from "class-validator";
+import { HistoricalVolume } from "../../model";
+import { getTimelineParams } from "./common";
+
+@ObjectType()
+export class TotalVolume {
+  @Field(() => String, { nullable: false })
+  date!: string;
+
+  @Field(() => BigInt, { nullable: false })
+  totalVolume!: bigint;
+
+  constructor(props: Partial<TotalVolume>) {
+    Object.assign(this, props);
+  }
+}
+
+@InputType()
+export class TotalVolumeInput {
+  @Field(() => Int, { nullable: false })
+  @Min(1)
+  intervalMinutes!: number;
+
+  @Field(() => String, { nullable: true })
+  @IsDateString()
+  dateFrom?: string;
+
+  @Field(() => String, { nullable: true })
+  @IsDateString()
+  dateTo?: string;
+}
+
+@Resolver()
+export class TotalVolumeResolver {
+  constructor(private tx: () => Promise<EntityManager>) {}
+
+  @Query(() => [TotalVolume])
+  async totalVolume(
```

Comment:
@andresmechali how about adding a `Currency` parameter here?

Reply:
Sure, I can do that. If it's ok I will include it in the next PR along with the decimal conversion.